typescript/index.ts

```diff
@@ -0,0 +1,48 @@
+export interface Workflow {
+  slug: WorkflowSlug;
+  name: string;
+  command: string;
+  tags?: [string];
+  description?: string;
+  arguments?: [Argument];
+  source_url?: string;
+  author?: string;
+  author_url?: string;
+  shells?: [Shell];
+}
+
+export interface Argument {
+  name: string;
+  description?: string;
+  default_value?: string;
+}
+
+export const enum Shell {
+  Bash,
+  Fish,
+  Zsh,
+}
```

Comment:
Enums are numeric by default in typescript. I'm wondering if we could use a string enum here or define a `to_string` function that will make it more useful for commands.dev

Reply:
done